dashboard/application.py

```diff
@@ -18,7 +18,9 @@ def __init__(self, table):
         self.tablename = table.tablename
 
     def url_path_for(self, name: str, **path_params: str):
-        path_params.pop("tablename", None)
+        tablename = path_params.pop("tablename", None)
```

Comment:
The most important change I think is here.
Before this, we'd only tested one datasource with dashboard, and when I added the second model in test, it still returned urls for the first datasource. So the second one was always ignored.

I'm dynamically creating name of endpoints and accessing them now. Not sure if it's the best approach, but works for now.